nextcloudappstore/core/models.py

```diff
@@ -506,16 +506,11 @@ def is_compatible(self, platform_version, inclusive=False):
 
     @property
     def is_unstable(self):
-        return (
-            self.is_nightly
-            or "-dev" in self.version
-            or "-a" in self.version
-            or "-alpha" in self.version
-            or "-b" in self.version
-            or "-beta" in self.version
-            or "-rc" in self.version
-            or "-RC" in self.version
-        )
+        if self.is_nightly:
+            return true
+
+        version = Version(self.version)
+        return version.prerelease
```

Comment:
does this need bool casting @oleksandr-nc

Reply:
Yes, should be `return bool(Version(self.version).prerelease)`